docs/installation/mac.md

```diff
@@ -61,7 +61,42 @@ redis-cli ping
 
 DEV requires Elasticsearch version 7 or higher.
 
-We recommend installing from archive on Mac. The following directions were
+You have the option of installing Elasticsearch with Homebrew of through an
+archive. We recommend installing from archive on Mac.
```

Comment:
I would have assumed that we recommend installing via brew on a Mac. It's a lot more seamless and brew is pretty much the default?

Reply:
I thought about changing this, but since we weren't able to make it work until now I don't know if we want to change the recommendation. I agree that it's easier to install, bar the two possible errors, because it can be updated "automatically"

Reply:
I'm also checking if I can report these problems upstream to ES so they might update the brew formula files

Reply:
Hmm I see your point, if it's not easy for everyone to get installed easily the first time let's leave it like this 👍